src/vs/gitpod/browser/workbench/workbench.ts

```diff
@@ -1,6 +1,6 @@
+/* eslint-disable header/header */
```

Comment:
I think you can configure it in root eslint file to exlude some paths

Reply:
yeah but I really don't mind this as there are just 3 files (gitpod extensions doesn't complain) plus it's less changes to maintain :)
Do you prefer to configure it globally?

Reply:
as you think it is better, was thinking it would be cleaner in long run